packages/@aws-cdk/pipelines/lib/lambda/index.py

```diff
@@ -0,0 +1,35 @@
+import boto3
```

Comment:
I don't mind Python, per se, but why not TypeScript?

Reply:
I saw that `s3-deployments` was in python so used that as an artifact

Reply:
That's a historical accident ;)

Reply:
If python is something we are moving away from, I dont mind changing this to typescript

Reply:
Yeah I'd prefer TypeScript.